src/main/java/frc/robot/RobotContainer.java

```diff
@@ -33,7 +33,7 @@
   public RobotContainer() {
     chassisSubsystem_ = new ChassisSubsystem();
     armSubsystem_ = new ArmSubsystem();
-    intakeSubsystem = new IntakeSubsystem();
+    intakeSubsystem = new IntakeSubsystem()
```

Comment:
Missing a semi-colon
```suggestion
    intakeSubsystem = new IntakeSubsystem(); /* added the semi-colon */
```